loki/transformations/single_column/revector.py

```diff
@@ -109,6 +109,8 @@ class BaseRevectorTransformation(Transformation):
         to define the horizontal data dimension and iteration space.
     """
 
+    _reduction_match_pattern = r'reduction\([\+\w:0-9 \t]+\)'
```

Comment:
If I'm reading this correctly, then `+` has been added as a valid operator here. Shouldn't we Add support for all standard-compliant operators?
https://www.openacc.org/sites/default/files/inline-images/Specification/OpenACC-3.3-final.pdf#subsection.2.5.15
Also, we're currently missing variables with underscores...

A not very strict but improved pattern for this could look like the following
```suggestion
    _reduction_match_pattern = r'reduction\([\+\*\.\w:0-9 \t]+\)'